test/DynamoCoreWpfTests/SerializationTests.cs

```diff
@@ -580,17 +580,26 @@ private void DoWorkspaceOpen(string filePath)
 
         }
 
-        private static string SaveJsonTempWithFolderStructure(DynamoViewModel viewModel, string filePath, JObject jo)
-        {   
+        /// <summary>
+        /// Copy test file to specified folder while 
+        /// maintaining original directory structure
+        /// and assigning file path as workspace name.
+        /// </summary>
+        /// <param name="filePath">original test file path</param>
+        /// <param name="jo">test json object</param>
+        private static void SaveJsonTempWithFolderStructure(string filePath, JObject jo)
+        {
             // Get all folder structure following "\\test"
-            var expectedStructure = filePath.Split(new string[] { "\\test" }, StringSplitOptions.None).Last();
+            int subStringIndex = filePath.IndexOf("\\test") + 5;
+            var expectedStructure = filePath.Substring(subStringIndex);
```

Comment:
@ramramps this fixes the issues where the path was getting split multiple times when the file name also began with` \\test`.  Is there a better way of doing this?  I need to split the file path string at the first occurrence of `\\test` and only keep the later half of the split.  The `+ 5` is awkward accounting for the length of `\test` since the index of the first character is returned.

Reply:
Only found good solutions for splitting by single characters

Reply:
Easiest would be to simply pass in the path to the root test directory.

Reply:
I think it's even a static member of the class, no passing required.

Reply:
ahh `SerializationTests.TestDirectory`, thanks!